### DISCUSSION TITLE
Passengers boarding

### DISCUSSION THREAD

**Discussion by emsss78** [flybywiresim/aircraft#6322] [Help]
Hello,

I noticed in the latest Developer version the passenger boarding is very slow...is it possible to add an option to make the boarding instant??

**Comment by pdeswardt** [comment]
It's already there - in the flypad - just set it to instant.

**Comment by emsss78** [comment]
On the EFB it's already on instant and that applies to fuel only...I meant the new boarding system in the MCDU (W/B) page.

**Reply by tracernz** [reply]
The UI is unfortunately a little inconsistent at the moment. The boarding speed option is under Settings -> Sim Options

**Reply by emsss78** [reply]
> The UI is unfortunately a little inconsistent at the moment. The boarding speed option is under Settings -> Sim Options

Ok thanks...I'll check it out

**Reply by emsss78** [reply]
It worked...thanks @tracernz

**Reply by f5vmr** [reply]
A similar issue is the problems of the passengers deplaning (deboarding). Clicking on the deplaning Icon seems to do nothing at first push then appears to reload them before emptying the plane, seems a little inconsistent, can you check the operation please?
